### PR TITLE
Plugwise Add update and removal information

### PR DESCRIPTION
**Pull request by CoMPaTech**
## Proposed change
Add information as per rules for the quality scale

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Added a "Data updates" section detailing refresh intervals for power, climate, and Stretch entities.
	- Introduced a "Removing the integration" section clarifying the integration removal process.
- **Documentation**
	- Enhanced clarity and usability of the Plugwise integration documentation with new content.

### CODE REVIEW

File: source/_integrations/plugwise.markdown [hunk 1]
```diff
@@ -57,6 +57,18 @@ For example, if you have an Adam setup with a Lisa named 'Living' and a Tom name
...
+
+The interval which the integration fetches data from the Smile depends on the device:
+
+- Power entities, such as the P1, will be refreshed every 10 seconds.
+- Climate entities will be refreshed every 60 seconds.
+- Stretch entities will be refreshed every 60 seconds.
+
+## Removing the integration
+
+This integration follows standard integration removal. No extra steps are required within Home Assistant or on your Plugwise devices.
```
Comment (frenck) [inline source/_integrations/plugwise.markdown:70]: Please include the snippet with the standard removal instructions (similar to the include econfig flow includes)

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @CoMPaTech 👍 

../Frenck

File: source/_integrations/plugwise.markdown (hunk 1 above)
Comment (CoMPaTech) [reply source/_integrations/plugwise.markdown:70]: Tnx for that - in the same light the configuration section indeed would also benefit from the includes - though to keep things small, will do that in a follow up PR.

Comment (CoMPaTech) [reply source/_integrations/plugwise.markdown:70]: Reference #36087

File: source/_integrations/.reolink.markdown.swp
Comment (frenck) [inline source/_integrations/.reolink.markdown.swp:1]: This file doesn't belong here I think?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/.reolink.markdown.swp
Comment (CoMPaTech) [reply source/_integrations/.reolink.markdown.swp:1]: Apologies for sneaking that in - thought I already removed it previously from the commit.

**Review by frenck — APPROVED** [review]
Thanks, @CoMPaTech 👍 

../Frenck

**Pull request merged**